nextflow.config

```
@@ -4,6 +4,8 @@ params{
   SALMON_CONTAINER = 'quay.io/biocontainers/salmon:1.5.2--h84f40af_0'
   ALEVINFRY_CONTAINER = 'quay.io/biocontainers/alevin-fry:0.4.1--h7d875b9_0'
   SCPCATOOLS_CONTAINTER = 'ghcr.io/alexslemonade/scpca-tools'
```

If I made this mistake elsewhere, we should correct it there as well!
```suggestion
  SCPCATOOLS_CONTAINER = 'ghcr.io/alexslemonade/scpca-tools'
```